views/layouts/layout.html

```diff
@@ -38,6 +38,9 @@
 		{{/if}}
 		<script async custom-element="amp-list" src="https://cdn.ampproject.org/v0/amp-list-0.1.js"></script>
 		<script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.1.js"></script>
+		{{#if unfurlBrightcove}}
```

Comment:
This isn't quite valid but it will work fine until we start getting new videos through ES. It needs to detect if the article has a video. We should probably use an n-ui style handlebars block to let the article pipeline add scripts to this section when it detects features.